server/migrations/20211225065548-refactor_notifications.js

```diff
@@ -0,0 +1,168 @@
+// eslint-disable-next-line max-len
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      // renames the current Notifications table to Old_Notifications
+      await queryInterface.sequelize.query(
```

Comment:
is there a reason you don't use the sequelize interface methods to do simple queries like this? 

simply: `sequelize.renameTable(old, new, options);`

Reply:
Wrote all the queries raw to test them out in DataGrip and didn't convert - I'll convert any that can be.